lib/cfn-nag/custom_rules/LambdaFunctionCloudWatchLogsRule.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/util/wildcard_patterns'
+require 'cfn-nag/violation'
+require_relative 'base'
+
+class LambdaFunctionCloudWatchLogsRule < BaseRule
+  IAM_CREATEGROUP_PATTERNS = wildcard_patterns('CreateLogGroup').map! { |x| 'logs:' + x } + ['*']
```

Comment:
the magic that looks at the role itself.... consider breaking it out into a separate class.  might make the actual rule logic easier to read?  your call though.

Reply:
Same as above.  This could be moved into cfn-model policy.

Reply:
Updated rule logic to utilize enhanced cfn-model parsing.